autokeras/hypermodel/block.py

```diff
@@ -538,6 +540,29 @@ def build(self, hp, inputs=None):
                                                         default=128)
         if pretraining:
             # TODO: load from pretrained weights
+            embedding_matrix = np.zeros((input_node.shape[1] + 1, embedding_dim))
+            if pretraining == 'glove':
+                dirname = 'glove.6B.100d.txt'
+                origin = 'http://nlp.stanford.edu/data/glove.6B.zip'
```

Comment:
Put them into const.py and rename the variable with all CAPITAL letters and with glove inside.